source/squidplus-proxy/stroomContent/SQUID-PLUS-XML.Folder.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.1" encoding="UTF-8"?>
```

Comment:
We need to agree a convention for the folder structure in stroom.  I think the system specific content folders should be bundled into one or more parent folders to keep things tidy.